test/jest/victory-bar/bar.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import * as d3Scale from "d3-scale";
```

Comment:
As an FYI, PR #2204 is going to bring in our new vendoring approach to d3 libraries. If that one lands first, you'll need to update this (and similar imports except `d3-voronoi`) to:

```suggestion
import * as d3Scale from "victory-vendor/d3-scale";
```

If this PR lands first, then no changes are needed!